fe/fe-core/src/main/java/com/starrocks/sql/ast/MVColumnItem.java

```diff
@@ -57,23 +58,17 @@ public class MVColumnItem {
     private boolean isAllowNull;
     private boolean isAggregationTypeImplicit;
     private Expr defineExpr;
-    private final String baseColumnName;
+    private Set<String> baseColumnNames;
```

Comment:
why it is a set?

Reply:
We can support more complex expressions later, eg, a+b as Sync MV's column.